README.md

```diff
@@ -22,6 +22,18 @@ docker compose run django python manage.py init
 # Wait a little, then check that https://localhost/oapif/collections/signalo_core.pole/items works from your browser
 ```
 
+## Authentication & permissions
+
+By default the viewsets under `signalo/core` use the `AllowAny` permissions class. You can override these app-level permissions for particular viewsets when registering their corresponding models. (Refer to https://www.django-rest-framework.org/api-guide/permissions/#api-reference for permission classes). Example:
+
+    models.py
+    ---------
+    @register_oapif_viewset(
+        custom_viewset_attrs={
+            "model_to_viewset_permissions": (permissions.DjangoModelPermissionsOrAnonReadOnly,)
+        }
+    )
```

Comment:
The intended usage is:
```
@register_oapif_viewset(
    custom_viewset_attrs={
        "permission_classes": (permissions.DjangoModelPermissionsOrAnonReadOnly,)
    }
)
```

(the keys are directly the DRF class attributes of the viewset, so we need less specific logic (see comment below in the decorator)

Also in the snippet, it helps to include the imports and a (partial) model definition (at least with the geom field), otherwise not very clear to what the decorator applies.